src/python/pants/core/goals/update_build_files.py

```diff
@@ -474,13 +474,13 @@ def determine_renamed_field_types(
 ) -> RenamedFieldTypes:
     target_field_renames: DefaultDict[str, dict[str, str]] = defaultdict(dict)
     for tgt in target_types.types:
-        field_renames = target_field_renames[tgt.alias]
-        if tgt.deprecated_alias is not None:
```

Comment:
You are no longer considering the deprecated alias of targets...?

Reply:
I wasn't sure why you had that snippet - we already have a dedicated rule for deprecated target names. This is only about deprecated field names. 

Ohhhh wait nevermind, this is so that you also fix if a user is still using deprecated targets with a deprecated field? Got it. I'll add that back and maybe expand the test.